backend/clutch-config.yaml

```diff
@@ -24,6 +24,16 @@ modules:
   - name: clutch.module.k8s
   - name: clutch.module.kinesis
 services:
+  - name: clutch.service.db.postgres
```

Comment:
Should we keep this in the default config?

Reply:
ah yeah sorry I usually comment on this, but ill remove the config changes once the review is complete.